### PR TITLE
update go test workflow

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -112,71 +112,17 @@ workflows:
     - test_with_no_podfile_lock
     - test_with_gemfile
     - test_without_gemfile
-    - test_podfile_at_root
+    # - test_podfile_at_root
```
Comment (trapacska) [inline bitrise.yml]: please remove commented-out parts

Comment (trapacska) [reply bitrise.yml]: non-blocking: format version is too old

File: main.go [hunk 2]
```diff
@@ -329,8 +331,12 @@ func main() {
 
 		cmd.SetDir(podfileDir)
 
-		if err := cmd.Run(); err != nil {
-			failf("Command failed, error: %s", err)
+		if out, err := cmd.RunAndReturnTrimmedCombinedOutput(); err != nil {
+			if errorutil.IsExitStatusError(err) {
```
Comment (trapacska) [inline main.go]: errors.Wrap would be a shorter solution:

```
if out, err := cmd.RunAndReturnTrimmedCombinedOutput(); err != nil {
  failf(errors.Wrap(err, out))
}
```

File: README.md [hunk 3]
```diff
@@ -76,4 +78,15 @@ in the [bitrise CLI repository](https://github.com/bitrise-io/bitrise/blob/maste
 
 ## Share your own Step
 
-You can share your Step or step version with the [bitrise CLI](https://github.com/bitrise-io/bitrise). Just run `bitrise share` and follow the guide it prints.
+You can share your Step or step version with the [bitrise CLI](https://github.com/bitrise-io/bitrise). If you use the `bitrise.yml` included in this repository, all you have to do is:
+
+1. In your Terminal / Command Line `cd` into this directory (where the `bitrise.yml` of the step is located)
+1. Run: `bitrise run test` to test the step
+1. Run: `bitrise run audit-this-step` to audit the `step.yml`
+1. Check the `share-this-step` workflow in the `bitrise.yml`, and fill out the
```
[the remark this answers is not in the document]
Comment (godrei) [reply README.md]: it is about sharing your own step, your own step generated by the step plugin will have that workflow

**Pull request merged**